eq-author/src/App/page/Design/QuestionPageEditor/index.js

```diff
@@ -3,21 +3,29 @@ import styled from "styled-components";
 import PropTypes from "prop-types";
 import CustomPropTypes from "custom-prop-types";
 import gql from "graphql-tag";
-
-import DeleteConfirmDialog from "components/DeleteConfirmDialog";
-
-import Page from "graphql/fragments/page.graphql";
+import { flowRight } from "lodash";
 
 import getIdForObject from "utils/getIdForObject";
+import withChangeUpdate from "enhancers/withChangeUpdate";
+import withEntityEditor from "components/withEntityEditor";
+import focusOnEntity from "utils/focusOnEntity";
+import withPropRenamed from "enhancers/withPropRenamed";
+import PageHeader from "../PageHeader";
 
 import AnswersEditor from "./AnswersEditor";
 import AnswerTypeSelector from "./AnswerTypeSelector";
-import MovePageModal from "./MovePageModal";
-import MovePageQuery from "./MovePageModal/MovePageQuery";
+import withUpdateQuestionPage from "./withUpdateQuestionPage";
+
+import withUpdateAnswer from "../answers/withUpdateAnswer";
```

Comment:
Could we move these next to this component or are they shared?

Reply:
Looks like the answerUpdate is shared between this and the properties panel.